test/internal/helpers/Player.java

```diff
@@ -235,10 +235,12 @@ public static Cleanups withItemInStash(final String itemName, final int count) {
   private static Cleanups addToList(final AdventureResult item, final List<AdventureResult> list) {
     var old = item.getCount(list);
     AdventureResult.addResultToList(list, item);
+    EquipmentManager.updateEquipmentLists();
```

Comment:
I like this too. Does this update available outfits? Nice!

Reply:
Yep!